### PR TITLE
feat: display cta banner when an un-deployed policy is selected

### PR DESCRIPTION
**Pull request by a-cordier**
see https://gravitee.atlassian.net/browse/APIM-1935

---
#### 🧪&nbsp;&nbsp;Gravitee.io Automatic Prerelease @gravitee/ui-policy-studio-angular
```
npm install @gravitee/ui-policy-studio-angular@7.26.0-lock-policy-studio-v4-36bd70a
```
```
yarn add @gravitee/ui-policy-studio-angular@7.26.0-lock-policy-studio-v4-36bd70a
```

---
#### 🧪&nbsp;&nbsp;Gravitee.io Automatic Prerelease @gravitee/ui-particles-angular
```
npm install @gravitee/ui-particles-angular@7.26.0-lock-policy-studio-v4-36bd70a
```
```
yarn add @gravitee/ui-particles-angular@7.26.0-lock-policy-studio-v4-36bd70a
```

---

### CODE REVIEW

File: ui-particles-angular/projects/ui-particles-angular/src/lib/gio-banner/gio-banner.component.scss [hunk 1]
```diff
@@ -23,6 +23,7 @@ $typography: map.get(gio.$mat-theme, typography);
 
 // Map for palette by type
 $types: (
+  cta: gio.$mat-error-palette,
```
[the remark this answers is not in the document]
Comment (a-cordier) [reply ui-particles-angular/projects/ui-particles-angular/src/lib/gio-banner/gio-banner.component.scss:26]: I asked myself, but the icon is different (and position is different as well). Maybe we can make the icon optional and use error for the banner.

[the remark this answers is not in the document]
Comment (a-cordier) [reply ui-particles-angular/projects/ui-particles-angular/src/lib/gio-banner/gio-banner.component.scss:26]: Screenshot 2023-07-04 at 14 12 52

File: ui-particles-angular/projects/ui-policy-studio-angular/src/lib/components/policies-catalog-dialog/gio-ps-policies-catalog-dialog.component.scss [hunk 2]
```diff
@@ -73,6 +74,21 @@ $typography: map.get(gio.$mat-theme, typography);
         }
       }
 
+      &__enterprise {
+        position: absolute;
+        top: 0;
+        right: 0;
+        padding: 8px 8px 8px 16px;
+        border-radius: 0 8px 0 38px;
+        background: var(--neutrals-dove-dove-dark-10, #e7e8ef);
```
Comment (ThibaudAV) [inline ui-particles-angular/projects/ui-policy-studio-angular/src/lib/components/policies-catalog-dialog/gio-ps-policies-catalog-dialog.component.scss:83]: I'm proposing a more clean css tomorrow

**Review by ThibaudAV — APPROVED** [review]

**Pull request merged**